commcare_connect/templates/microplanning/home.html

```
@@ -109,13 +102,48 @@ <h1 class="multi-line-clamp-2 text-2xl font-semibold text-white break-words hyph
-            <i class="fa-solid fa-trash"></i> {% translate "Clear Work Area Groups" %}
-          </button>
         {% endif %}
+        <div x-data="{open: false}" @click.outside="open = false">
+          <button class="button button-md outline-style" @click="open = !open">
+            <i class="mr-2 fa-solid fa-trash"></i>
+            {% translate "Clear Data" %}
+            <i class="ml-2 fa-solid" :class="open ? 'fa-angle-up' : 'fa-angle-down'"></i>
+          </button>
+          <div class="dropdown hidden" :class="{'hidden': !open}">
+            <ul class="dropdown-list">
+              <li class="dropdown-item {% if not show_clear_work_areas_btn %}disabled{% endif %}"
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)